group_variables.go

```diff
@@ -127,6 +128,7 @@ type CreateGroupVariableOptions struct {
 	Description      *string            `url:"description,omitempty" json:"description,omitempty"`
 	EnvironmentScope *string            `url:"environment_scope,omitempty" json:"environment_scope,omitempty"`
 	Masked           *bool              `url:"masked,omitempty" json:"masked,omitempty"`
+	Hidden           *bool              `url:"hidden,omitempty" json:"hidden,omitempty"`
```

Comment:
From the [docs](https://docs.gitlab.com/ee/api/group_level_variables.html#create-variable:~:text=variable%20is%20masked.-,masked_and_hidden,-boolean) it seems the field is called `masked_and_hidden`. Did `hidden` work for you?

Reply:
Same comment as https://github.com/xanzy/go-gitlab/pull/2065#pullrequestreview-2452420467